### PR TITLE
emacs: Define _snprintf for ucrt.

### PR DESCRIPTION
**Pull request by Biswa96**
Run make three times for unknown failures. At least it builds. Suggestions are welcomed!

### CODE REVIEW

File: mingw-w64-emacs/PKGBUILD [hunk 1]
```diff
@@ -69,6 +65,10 @@ build() {
   # --without-compress-install is needed because we don't have gzip in
   # the mingw binaries.
 
+  # Workaround: make fails with bootstrap-emacs.pdmp and emacs.pdmp
+  # for unknown reason.
+  make || true
+  make || true
   make
```
Comment (jeremyd2019) [inline mingw-w64-emacs/PKGBUILD:72]:
```suggestion
  make || make || make
```

??  I've had to do things like that before ☹️

**Comment by Biswa96** [comment]
Hope I am allowed to copy things from mingw-w64! Am I? @lhmouse

**Comment by lhmouse** [comment]
Yes of course.

**Comment by Biswa96** [comment]
Closed! emacs crashes in UCRT. May need special treatment.

**Pull request closed without merging**